WooCommerce/Classes/ViewRelated/Orders/Order Details/Customer Section/Billing Information/Cells/BillingAddressTableViewCell.swift

```diff
@@ -114,7 +114,6 @@ private extension BillingAddressTableViewCell {
 
     func configureEditButton() {
         editButton.applyIconButtonStyle(icon: .pencilImage)
-        editButton.contentEdgeInsets = UIEdgeInsets(top: 12, left: 12, bottom: 12, right: 12)
```

Comment:
Thanks for the response. Just a clarification: the edget insets being ignored is a bug due to the deprecation of the API. Instead of removing the line of code to get rid of the warning, we should try to apply a fix to keep the edge insets if we want to follow the original intention. In case you missed my suggestion above, here's a suggested snippet for the fix:

```Swift
var configuration = UIButton.Configuration.plain()
configuration.contentInsets = .init(top: 12, leading: 12, bottom: 12, trailing: 12)
editButton.configuration = configuration
```

I believe we should do something similar to other buttons as well.